release-notes.rst

```diff
@@ -1,14 +1,25 @@
 Release Notes
 =============
 
+Version 0.2.0b1
+_____________
+- hopeit.aws.s3 
+
+   - Updated to be compatible with `hopeit.engine` version 0.25.0.
+   
+   BREAKING CHANGES
+   ================
+
+   - This version is not backward compatible with previous version of the `hopeit.engine`.
```

Comment:
"This version is not backwards compatible with previous versions of `hopeit.engine`"